Classes/Global State Explorers/FLEXWebViewController.m

```diff
@@ -8,10 +8,11 @@
 
 #import "FLEXWebViewController.h"
 #import "FLEXUtility.h"
+#import <WebKit/WebKit.h>
 
-@interface FLEXWebViewController () <UIWebViewDelegate>
+@interface FLEXWebViewController () <WKNavigationDelegate>
```

Comment:
Do we even use this class somewhere?

Reply:
It's `#if`-ed in the battle test menu. I don't know if anyone uses it, it seems like if anyone would it would be Art. We could have removed it but honestly it was easy enough to port that I think it would take longer to figure out if it truly is unused.